src/System.Management.Automation/engine/Modules/NewModuleManifestCommand.cs

```diff
@@ -1197,6 +1217,9 @@ private void BuildPrivateDataInModuleManifest(StringBuilder result, StreamWriter
                 BuildModuleManifest(result, "ProjectUri", Modules.ProjectUri, ProjectUri != null, () => QuoteName(ProjectUri), streamWriter);
                 BuildModuleManifest(result, "IconUri", Modules.IconUri, IconUri != null, () => QuoteName(IconUri), streamWriter);
                 BuildModuleManifest(result, "ReleaseNotes", Modules.ReleaseNotes, !string.IsNullOrEmpty(ReleaseNotes), () => QuoteName(ReleaseNotes), streamWriter);
+                BuildModuleManifest(result, "Prerelease", Modules.Prerelease, !string.IsNullOrEmpty(Prerelease), () => QuoteName(Prerelease), streamWriter);
```

Comment:
Perhaps we could use `nameof()` in `BuildModuleManifest()` like `nameof(Prerelease)`.
It is question only.
/cc  @daxian-dbw

Reply:
That would be good :)

Reply:
@pougetat Could you please make the change in all calls of `BuildModuleManifest()`?

Reply:
done

Reply:
Fill free to pull new PR or open new tracking issue.